e2e/playwright/tests/todo_plugin.spec.ts

```diff
@@ -6,35 +6,114 @@
 // - [*] indicates an assertion (e.g. * Check the title)
 // ***************************************************************
 
-import {expect, test} from '@e2e-support/test_fixture';
-import SlashCommandSuggestions from 'support/components/slash_commands';
-import {fillMessage, getTodoBotDMPageURL} from 'support/utils';
+import {expect, test} from "@e2e-support/test_fixture";
+import SlashCommandSuggestions from "support/components/slash_commands";
+import {
+    fillMessage,
+    getBotDMPageURL,
+    getLastPost,
+    getTeamName,
+    postMessage,
+} from "support/utils";
+
+const botUserName = "todo";
+let teamName = "";
+
+test.beforeAll(async ({ pw }) => {
+  const { adminClient, adminUser } = await pw.getAdminClient();
+  if (adminUser === null) {
+    throw new Error("can not get adminUser");
+  }
+  if (teamName === "") {
+    teamName = await getTeamName(adminClient, adminUser.id);
+  }
+});
+
+test.beforeEach(async ({ page }) => {
+  const dmURL = await getBotDMPageURL(teamName, botUserName);
+  await page.goto(dmURL, { waitUntil: "load" });
```

Comment:
I'm thinking we should clear the bot DMs before each test, since we're using the same channel for each test. We can use the function defined here for this: https://github.com/mattermost-community/mattermost-plugin-todo/blob/b8dff94f7bebfcb7f81f36f588f110e08e34ef9f/e2e/playwright/support/utils.ts#L43